src/util/helpers.ts

```diff
@@ -107,6 +108,7 @@ const devConstants = {
     BTC_SEGWIT_NATIVE_V0_PREFIX: 'tb',
     BTC_LEGACY_VERSION: 0x6F,
     BTC_WRAPPED_SEGWIT_VERSION: 0xC4,
+    POAP_CLAIM_LOCAL_URL: 'http://localhost:5001/lattice-poap/us-central1/validate'
```

Comment:
Iz cruft 🧐

Reply:
Do we want to make it configurable for local/remote? Or is there somewhere else we should keep that info?

Reply:
The way I organized this repo before was to overload a few local constants at init time. Not sure if you're doing that here but it would just re-use the other constant and replace it with this value on a dev build

Reply:
In this case, we don't want to _always_ use that in development. I'll remove it here and document it in the other repo. Because you'd only use it here if you were working on that one.